src/RuleSet/Loader/Composer/Package.php

```diff
@@ -42,7 +42,7 @@ public static function fromArray(array $package)
      */
     public function generatePackageKey()
     {
-        return str_replace('/', '_', $this->name).'_'.$this->version;
+        return str_replace(array('/', '\\', ':'), '_', $this->name).'_'.$this->version;
```

Comment:
Maybe it would now be a good idea to make sure that there won't be several subsequent underscores in the generated key?